### PR TITLE
Update  some mission "memo" information of quests-tcn file

### PR DESCRIPTION
**Pull request by skysumbra**
Update A1 to A93 mission "memo" information

### CODE REVIEW

**Comment by skysumbra** [comment]
Commit 3rd  version add factory and drill mission description.

**Review by antest1 — CHANGES_REQUESTED** [review]
Hi skysumbra! first of all, thanks for your great contributions! 🙏🙏
I marked lines that need to be fixed before merge, so please check and make new commit!

File: files/quests-tcn.json [hunk 1]
```diff
@@ -97,411 +110,458 @@
...
-		"desc": "以輕巡洋艦「天龍」「龍田」以及另外2艘艦為基幹戰力新編成4艘以上的「第十八戰隊」"
+		"desc": "以輕巡洋艦「天龍」「龍田」以及另外2艘艦為基幹戰力新編成4艘以上的「第十八戰隊」",
+		"memo": "※開發資材*2"
 	},
 	"154": {
 		"code": "A52",
 		"name": "海上突入部隊編成",
-		"desc": "以戰艦「比叡」「霧島」輕巡「長良」驅逐艦「曉」「雷」「電」編成海上突入部隊"
+		"desc": "以戰艦「比叡」「霧島」輕巡「長良」驅逐艦「曉」「雷」「電」編成海上突入部隊",
+		"memo" "開發資材*2，高速修復材*2 "
```
[the remark this answers is not in the document]
Comment (skysumbra) [reply files/quests-tcn.json]: Sorry, i update file again.

File: files/quests-tcn.json [hunk 2]
```diff
@@ -1842,148 +1842,153 @@
...
+		"desc": "航路護衛任務:以兩艘航空戰艦(或者補給艦)+任意船艦*4為核心編成艦隊，在1-6實施運輸船隊護衛作戰並成功抵達目的地2次",
+		"memo": "高速修復*4 和 洋上補給"
 	},
 	"862": {
 		"code": "Bq4",
-		"name": "(季任) 實施前線航空偵察",
-		"desc": "偵察任務:以1艘水上機母艦和2艘輕巡洋艦+任意船艦*3為核心編成偵察艦隊，在6-3展開，反覆實施航空偵察作戰「K作戰」!進一步捕捉該海域敵軍艦隊，擊破敵軍戰力，需要BOSS戰A勝及以上兩次"
+		"name": "(季常) 實施前線航空偵察",
+		"desc": "偵察任務:以1艘水上機母艦和2艘輕巡洋艦+任意船艦*3為核心編成偵察艦隊，在6-3展開，反覆實施航空偵察作戰「K作戰」!進一步捕捉該海域敵軍艦隊，擊破敵軍戰力，需要BOSS戰A勝及以上兩次",
+		"memo" "開發資材*4 和 改修資材*4"
```
[the remark this answers is not in the document]
Comment (skysumbra) [reply files/quests-tcn.json:1865]: Update new file, thanks antest1

[the remark this answers is not in the document]
Comment (antest1) [reply files/quests-tcn.json:1865]: oh seems you forgot to update this part, can you check it again?

[the remark this answers is not in the document]
Comment (skysumbra) [reply files/quests-tcn.json:1865]: sorry, let me upload again.

**Comment by skysumbra** [comment]
some "memo" column colon lost in [5ab8bba](https://github.com/antest1/kcanotify-gamedata/pull/3/commits/5ab8bbad3de81616c3af931529878597abd422fa) add

**Review by antest1 — APPROVED** [review]

**Pull request merged**